config/jobs/README.md

```diff
@@ -1,9 +1,213 @@
+- [`preset-aws-credentials: "true"`] ensures the prowjob has AWS credentials
+  for kops tests in a well known location, with an env var pointint to it
+
+## Job Examples
+
+A presubmit job named "pull-community-verify" that will run against all PRs to
+kuberentes/community's master branch. It will run `make verify` in a checkout
+of kubernetes/community at the PR's HEAD. It will report back to the PR via a 
+status context named `pull-kubernetes-community`. Its logs and results are going 
+to end up in GCS under `kubernetes-jenkins/pr-logs/pull/community`. It's using
+the `gcloud-in-go` image, which isn't recommended, but works for now.
+
+```yaml
+presubmits:
+  kubernetes/community:
+  - name: pull-community-verify  # convention: (job type)-(repo name)-(suite name)
+    branches:
+    - master
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - --scenario=execute
+        - --
+        - make
+        - verify
+```
+
+ci-test-infra-triage: periodic, runs a container, doesn't need a repo
+
+A periodic job named "periodic-cluster-api-provider-aws-test-creds" that will
+run every 4 hours against kubernetes-sigs/cluster-api-provider-aws's master
+branch. It will run `./scripts/ci-aws-cred-test.sh` in a checkout of the repo
+located at `sigs.k8s.io/cluster-api-provider-aws`. The presets it's using will
+ensure it has aws credentials and aws ssh keys in well known locations. Its 
+logs and results are going to end up in GCS under `kubernetes-jenkins/logs/periodic-cluster-api-provider-aws-test-creds`.
+It's using the `kubekins-e2e` image which isn't recommended
+
+```yaml
+periodics:
+- name: periodic-cluster-api-provider-aws-test-creds
+  decorate: true
+  interval: 4h
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-aws
+    base_ref: master
+    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190618-b782256-1.15
+      command:
+      - "./scripts/ci-aws-cred-test.sh"
+```
+
+## Adding or Updating Jobs
+
+1. Find or create the prowjob config file in this directory
+    - In general jobs for `github.com/org/repo` use `org/repo/filename.yaml`
+    - For kubernetes/kubernetes we prefer `kubernetes/sig-foo/filename.yaml`
+    - Ensure `filename.yaml` is unique across the config subdir; prow uses this as a key in its configmap
+1. Ensure an [`OWNERS`](https://go.k8s.io/owners) file exists in the directory for job, and has appropriate approvers/reviewers
+1. Write or edit the job config (please see [how-to-add-new-jobs](/prow/jobs.md#how-to-configure-new-jobs))
```

Comment:
Could you please also make `/prow/jobs.md` link back to this file with a disclaimer that it is specific to prow.k8s.io?  I think this doc would be useful for admins and users of other Prow instances too.

Reply:
done